x-pack/plugins/security_solution/public/management/pages/policy/view/protection_updates/hooks/use_post_protection_updates_note.ts

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { useMutation, useQueryClient } from '@tanstack/react-query';
+import { getProtectionUpdatesNoteQueryKey } from './use_get_protection_updates_note';
+import { useKibana } from '../../../../../../common/lib/kibana';
+import { resolvePathVariables } from '../../../../../../common/utils/resolve_path_variables';
+import { PROTECTION_UPDATES_NOTE_ROUTE } from '../../../../../../../common/endpoint/constants';
+
+interface UseProtectionUpdatesNote {
```

Comment:
nit: I'd avoid starting an interface with `Use`, maybe `ProtectionUpdatesNoteParams` would be better here

Reply:
Changed